config/services.yaml

```diff
@@ -2,3 +2,19 @@ parameters:
   database.name: '%env(FORK_DATABASE_NAME)%'
   kernel.secret: '%env(APP_SECRET)%'
   fork.is_installed: true
+
+services:
```

Comment:
Is dat niet iets wat default in core zou mogen? Althans de eerste 3 lijnen?

Reply:
Alles wordt al autoloaded ingeladen als een module is geinstaleerd. Dit is specifiek gedaan omdat we deze nodig hebben in onze security.yml